github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -484,6 +484,55 @@ presubmits:
           hostPath:
             path: /dev/vfio/
             type: Directory
+
+  - name: pull-kubevirt-e2e-kind-k8s-1.17.0-ipv6
+    skip_branches:
+    - release-\d+\.\d+
+    annotations:
+      fork-per-release: "true"
+    always_run: false
+    optional: true
+    decorate: true
+    decoration_config:
+      timeout: 7h
+      grace_period: 5m
+    max_concurrency: 10
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror: "true"
+      preset-shared-images: "true"
+    spec:
+      nodeSelector:
+        type: bare-metal-external
```

Comment:
Is this needed? Was not it just for SR-IOV?

Reply:
Let's go step by step, first we make it work, then we test with sriov.

Reply:
Thays boy answering my question. Is baremetal needed now? Can we disable it to save some traffic on those precious machines? Since this is just kind, I'd assume it can run in a VM just fine.

Reply:
provider is kind, but kubevirt needed nested virtualization.

Reply:
Ack